src/sklearndf/wrapper/_wrapper.py

```diff
@@ -1092,7 +1094,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     @property
     def labels_(self) -> pd.Series:
```

Comment:
I think this one is missing `@fitted_only` decorator.

Reply:
Indeed it is!